app/src/test/java/com/appunite/loudius/ui/reviewers/ReviewersViewModelTest.kt

```diff
@@ -237,7 +321,9 @@ class ReviewersViewModelTest {
                 viewModel.onAction(ReviewersAction.Notify("user1"))
                 viewModel.onAction(ReviewersAction.OnSnackbarDismiss)
 
-                assertNull(viewModel.state.snackbarTypeShown)
+                expectThat(viewModel.state)
+                    .get(ReviewersState::snackbarTypeShown)
+                    .isNull()
```

Comment:
**Suggestion** That will fit in one line.

Reply:
as above